fmriprep/interfaces/patches.py

```diff
@@ -77,3 +79,26 @@ def _run_interface(self, runtime):
                 sleep(randint(start + 4, start + 10))
 
         return runtime
+
+
+class MRICoregInputSpec(fs.registration.MRICoregInputSpec):
+    reference_file = File(
+        argstr="--ref %s",
+        desc="reference (target) file",
+        copyfile=False,
+    )
+    subject_id = traits.Str(
+        argstr="--s %s",
+        position=1,
+        requires=["subjects_dir"],
+        desc="freesurfer subject ID (implies ``reference_mask == "
+        "aparc+aseg.mgz`` unless otherwise specified)",
+    )
+
+
+class MRICoreg(fs.MRICoreg):
+    """
+    Patched that allows setting both a reference file and the subjects dir.
+    """
+
+    input_spec = MRICoregInputSpec
```

Comment:
this patch is needed to include the reference file + subject dir/id information in the outputted LTA